packages/opentelemetry/README.md

```diff
@@ -0,0 +1,4 @@
+
+## @dotcom-reliability-kit/opentelemetry
+
+This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
```

Comment:
**suggestion:** I don't think we need a full README in this PR but we could add the description in as well as a note to say this package is experimental:

```suggestion
An [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) client that's preconfigured for drop-in use in FT apps. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).

> [!WARNING]<br />
> This package is in beta and hasn't been tested extensively in production yet. Feel free to use, and any feedback is greatly appreciated.
```